server/src/servershot.js

```diff
@@ -18,8 +18,8 @@ class Shot extends AbstractShot {
     let toInsert = [];
     for (let clip of clips) {
       if (clip.image) {
-        if (! clip.isDataUrl()) {
-          toInsert.push(clip);
+        if (clip.isDataUrl()) {
+          toInsert.push(clip.id);
```

Comment:
Plus this made for double doh (made figuring out what the problem was extremely difficult)